packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -18,14 +18,19 @@ import {
   InternalMonitoring,
   callMonitored,
   createHandlingStack,
+  Omit,
+  includes,
 } from '@datadog/browser-core'
 import { RumEventDomainContext } from '../domainContext.types'
 import { CommonContext, User, ActionType } from '../rawRumEvent.types'
 import { RumEvent } from '../rumEvent.types'
 import { buildEnv } from './buildEnv'
 import { startRum } from './startRum'
 
-export interface RumInitConfiguration extends InitConfiguration {
+const ignoredConfigurationOptions = ['publicApiKey', 'datacenter'] as const
+type IgnoredConfigurationOptions = typeof ignoredConfigurationOptions[number]
```

Comment:
```suggestion
const deprecatedConfigurationOptions = ['publicApiKey', 'datacenter'] as const
type DeprecatedConfigurationOptions = typeof deprecatedConfigurationOptions[number]
```

Reply:
I've thought about it. But to me it's not really the `deprecated` options. It's the options we no logger support. 
At some point, we might have some other RUM deprecated options which won't be in this list. Won't it be a bit confusing?

Reply:
what difference do you see between deprecated options and options that we no longer support?
Do you have an example scenario where a deprecated options should not be in this list?

Reply:
To me, an option marked as `deprecated` is here to warn the user that the option is going to be removed at some point.
It also indicate, what's the new way to do what we want. 
For example we might have 
```js
export interface RumInitConfiguration extends Omit<InitConfiguration, IgnoredConfigurationOptions> {
  applicationId: string
  /**
   * @deprecated use XXXX instead
   */
  beforeSend?: (event: RumEvent, context: RumEventDomainContext) => void | boolean
}
```

Reply:
ok, so in our case they are removed from a user perspective but only deprecated in the code base since they are still handled by core. It depends from where we look.
IMO, `ignored` is not right because a reader can ask himself why are they ignored.
wdyt?

Reply:
I go with `droppedConfigurationOptions`